### PR TITLE
use a const for run button svg

### PR DESCRIPTION
**Pull request by madhur-tandon**
This follows the approach for what is done with `CLOSEBUTTON`

### CODE REVIEW

File: pyscriptjs/src/consts.ts
Comment (tedpatrick) [inline pyscriptjs/src/consts.ts:2]: empty id

Comment (madhur-tandon) [reply pyscriptjs/src/consts.ts]: removed, thanks!

**Comment by tedpatrick** [comment]
Do we use CLOSEBUTTON and RUNBUTTON outside of `<py-repl>`?

if no, why not keep them local to `<py-repl>` vs consts?

**Comment by madhur-tandon** [comment]
> Do we use CLOSEBUTTON and RUNBUTTON outside of `<py-repl>`?

`CLOSEBUTTON` is used in `exceptions.ts` while `RUNBUTTON` is used in `py-repl`

**Comment by madhur-tandon** [comment]
Re-running CI because of Timeout on one of Integration Tests.

**Comment by madhur-tandon** [comment]
> If RUNBUTTON is limited to `<py-repl>`... isn't that where it should live?

Yeah, makes sense. I guess I should move `CLOSEBUTTON` too I guess? WDYT?
But then, I don't think `_createAlertBanner` function which uses `CLOSEBUTTON` can be made into a plugin so it makes sense to keep that in `consts`

**Review by fpliger — APPROVED** [review]
👍

**Comment by tedpatrick** [comment]
I just want dependencies tracked against where they are used. Use your best judgment there and 🚢 it.

**Comment by FabioRosado** [comment]
Ah yeah that's my fault 😄  I introduced the `consts` file thinking that we could perhaps keep all of the things there, although maybe keeping them in the same file might be a good idea? 🤔

**Review by FabioRosado — APPROVED** [review]

**Comment by madhur-tandon** [comment]
Thanks! I have remove the `consts.ts` file introduced by @FabioRosado for now.
If needed, we can introduce common assets again later on. For now, they are located in the same file in which they are used.
Will merge when all tests pass.

**Comment by madhur-tandon** [comment]
@FabioRosado Seems like one of the tests were broken, I have fixed it, can you TAL?
Seems like it was introduced recently by you.

**Comment by FabioRosado** [comment]
> @FabioRosado Seems like one of the tests were broken, I have fixed it, can you TAL?
> Seems like it was introduced recently by you.

Oh! I wonder if this is caused by me merging #924, after this PR we are setting the text into `textContent` but the tests are looking at `inner_html` which will escape the `< >`. 


I wonder why it didn't break on that other PR 🤔  I think if we look at `text_content` instead it should fix the issue. Want me to open a quick fix for it?

**Comment by madhur-tandon** [comment]
> Want me to open a quick fix for it?

I will push some commits fixing it here, but will request you for review again if that's okay.

**Comment by FabioRosado** [comment]
Apologies for the ping, I ran the tests in main and it passes, but pulling this branch it fails. Did you rebase main into the branch? 🤔

**Comment by madhur-tandon** [comment]
> Did you rebase main into the branch? 🤔

Yeah, I did, so it shouldn't error out, right?

**Comment by madhur-tandon** [comment]
@FabioRosado actually, the CI fails on main too, so there is a problem. Will fix and push here only.

**Comment by FabioRosado** [comment]
> @FabioRosado actually, the CI fails on main too, so there is a problem. Will fix and push here only.
Oh wait... right I forgot to pull main 😬 and I can see what's happening, seems like I forgot to update the messages when merging that PR 😞

**Comment by madhur-tandon** [comment]
> seems like I forgot to update the messages when merging that PR 😞

If you have them locally, maybe you can open a quick PR and I will update this one after merging yours and rebasing, what say?

**Comment by FabioRosado** [comment]
> > seems like I forgot to update the messages when merging that PR 😞
> 
> If you have them ready locally, maybe you can open a quick PR and I will update this one after merging yours and rebasing, what say?

Yeah this is my bad sorry! Got #950 open to fix this 😄

**Pull request merged**
